Apps/Contoso.Android.Puppet/ModulePages/CrashesFragment.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.IO;
+using System.Threading;
```

Comment:
Not available in .net standard 1.x and did not have to migrate to .net standard 2.0 so the fix is tested in the xamarin.android app instead of forms.